src/DIRAC/Interfaces/API/Dirac.py

```diff
@@ -2536,3 +2545,128 @@ def getConfigurationValue(self, option, default):
         """Export the configuration client getValue() function"""
 
         return gConfig.getValue(option, default)
+
+    #############################################################################
+    def parseJobFromJDL(self, jdl: str):
```

Comment:
These APIs are already containing lots of code that should not necessarily be user-facing. I think this function is one of them.